sw-emulator/app/src/main.rs

```diff
@@ -350,9 +350,12 @@ fn main() -> io::Result<()> {
         // DWORD 00 - Flags
         cert[0] = flags.get();
         // DWORD 01 - 05 - IDEVID Subject Key Identifier (all zeroes)
-        // DWORD 06 - 07 - UEID / Manufacturer Serial Number
+        // DWORD 06 - 09 - UEID / Manufacturer Serial Number
         cert[6] = *args_ueid as u32;
         cert[7] = (*args_ueid >> 32) as u32;
+        cert[8] = (*args_ueid >> 64) as u32;
+        cert[9] = (*args_ueid >> 96) as u32;
+        cert[10] = 1; // UEID Type
```

Comment:
this should be cert[6] right?

Reply:
I defined UEID type as DWORD 10 in the fuse_idevid_cert_attr array:

    ManufacturerSerialNumber1 = 6,
    ManufacturerSerialNumber2 = 7,
    ManufacturerSerialNumber3 = 8,
    ManufacturerSerialNumber4 = 9,
    UeidType = 10,

I can make it DWORD 6 and move the UEID from DWORDs 7-10

Reply:
no need. was just confused whether this array was indexing cert blocks or fuse blocks.

Reply:
I see you did it anyway, that works.

Reply:
Updated fuse_idevid_cert_attr fields to make this more obvious:

    UeidType = 6,
    ManufacturerSerialNumber1 = 7,
    ManufacturerSerialNumber2 = 8,
    ManufacturerSerialNumber3 = 9,
    ManufacturerSerialNumber4 = 10,